README.md

```diff
@@ -11,43 +13,56 @@ Azuki is generic framework designed to assist with Behaviour Driven Specificatio
 ## Introduction
 
 BDD is a process used to drive development and testing at the level of a user story.
-BDD involves the creation of scenarios written from the point of view of the stakeholder, and which exemplify their requirements.
+BDD involves the creation of scenarios written from the point of view of the stakeholder, and which exemplify their
+requirements.
```

Comment:
🚫 Avoid adding new lines mid-sentence. The point of having one sentence per line is to keep the size of Git diffs to a minimum when making edits.

We _could_ consider semantic line breaks instead -- see e.g. https://sembr.org/ or https://rhodesmill.org/brandon/2012/one-sentence-per-line/ -- but that's a separate discussion and we shouldn't muddle the waters in this PR, which should only be concerned with American vs English spelling.